src/index.html

```diff
@@ -12,7 +12,40 @@
       href="styles/index.scss"
     />
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length-31 calendar--start-day-sun">
```

Comment:
The 'calendar--month-length-31' and 'calendar--start-day-sun' modifiers should be set dynamically based on the actual month length and start day. Hardcoding these values may not meet the requirements if the calendar needs to represent different months or start days.